Tests/SentryTests/SentryCrash/SentryThreadInspectorTests.swift

```diff
@@ -80,6 +81,27 @@ class SentryThreadInspectorTests: XCTestCase {
         wait(for: [expect], timeout: 10)
     }
 
+    func testGetCurrentThreadWithStackTrack_TooManyThreads() {
+        let expect = expectation(description: "Wait all Threads")
+        expect.expectedFulfillmentCount = 70
+
+        let sut = self.fixture.getSut(testWithRealMachineContextWrapper: true)
+
+        for _ in 0..<expect.expectedFulfillmentCount {
+            Thread.detachNewThread {
+                expect.fulfill()
+                while self.fixture.keepThreadAlive {
+                    Thread.sleep(forTimeInterval: 0.001)
+                }
+            }
```

Comment:
TIL; fancy way of starting multiple threads 👍

Reply:
DispatchQueue has a managing system that don't allow to run too many threads in parallel.